src/main/java/mate.academy/Main.java

```diff
@@ -8,9 +8,8 @@
 public class Main {
 
     public static void main(String[] args) {
-        // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
```

Comment:
It's good practice to ensure that the Injector class has a private constructor to prevent instantiation. Make sure that this is the case, as it's a common mistake.